R/filters.R

```diff
@@ -334,3 +337,51 @@ construct_tree <- function(data, id_column = 1, parent_id_column = 2,
 
   build_immediate_children(data[root_node, id_column])
 }
+
+get_programme_time_series_filters <- function(data) {
+  list(
+    list(
+      id = scalar("plot_type"),
+      column_id = scalar("plot"),
+      label = scalar(t_("INPUT_TIME_SERIES_FILTER_PLOT_TYPE")),
+      options = get_simple_filter(data, "plot")
+    ),
+    list(
+      id = scalar("area_level"),
+      column_id = scalar("area_level_label"),
+      label = scalar(t_("INPUT_TIME_SERIES_FILTER_AREA_LEVEL")),
+      ## TODO: Plot type and time step should use human readable name for label
```

Comment:
Did you mean to leave these TODOs? Any text fields are going to have to get translated in the front end anyway so human readability here isn't completely necessary - though it would be good if they were consistent with existing filters.

Reply:
Yeah, will fill these in with real labels when I implement the endpoint with real data. Might put code into naomi to manage that mapping of id to user readable name (this is how e.g. we are managing age group ID to human readable). 

Is doing the translations of this in the front end a new thing for this plot type? Does that mean we're going to be duping effort? Returning human readable labels elsewhere for filters is managed in hintr & naomi I thought?

Reply:
I might be misremembering, but I thought for the filters, the actual filter label, e.g. 'age', 'sex' was a translation key to be translated in the front end, but the options labels ('male' etc) were translated in hintr.

Reply:
Oh good, I think we're on the same page then. This comment is about the options. Have clarified that in the TODO.